fhir-server-test/src/test/java/com/ibm/fhir/server/test/ServerSpecTest.java

```diff
@@ -686,4 +687,94 @@ public void testConditionalUpdateObservation2() throws Exception {
                 "Input resource 'id' attribute must match the id of the search result resource");
         
     }
+    
+    // Test: retrieve Patient with _summary=true.
+    @Test(groups = { "server-spec" }, dependsOnMethods={"testCreatePatient"})
+    public void testReadPatientSummary() {
+        WebTarget target = getWebTarget();
+        Response response = target.path("Patient/" + savedPatient.getId().getValue())
+                .queryParam("_summary", "true")
+                .request(FHIRMediaType.APPLICATION_FHIR_JSON).get();
+        assertResponse(response, Response.Status.OK.getStatusCode());
+        Patient responsePatient = response.readEntity(Patient.class);
+        Coding subsettedTag =
+                Coding.builder().system(uri("http://terminology.hl7.org/CodeSystem/v3-ObservationValue")).code(Code.of("SUBSETTED")).display(string("subsetted")).build();
+        assertTrue(FHIRUtil.hasTag(responsePatient, subsettedTag));
```

Comment:
I know we are deprecating FHIRUtil is there an alternative in ModelSupport?

Reply:
no, we don't have such util Functions in ModelSupport yet, seems all tag helper functions are in FhirUtil only at present.